src/engine/test/assets/decoders/auditd.yml

```diff
@@ -0,0 +1,153 @@
+
+    - event.original: >-
+        $TYPE_FIELD $MSG_FIELD <_log.msg/quoted/ / msg=><_log.sub_msg/quoted/"/">
+
+    - event.original: >-
+        $TYPE_FIELD $MSG_FIELD<_log.msg/quoted/ / msg=><_log.sub_msg/quoted/'/'>
+
+    - event.original: >-
+        $TYPE_FIELD $MSG_FIELD<_log.msg/quoted/ / msg=><_log.sub_msg/quoted/"/">
+
+    - event.original: >-
+        $TYPE_FIELD $MSG_FIELD <_avc/quoted/avc:/for > <_log.msg>
+
+    - event.original: >-
+        $TYPE_FIELD $MSG_FIELD <_avc/quoted/avc:/for ><_log.msg>
+
+    - event.original: >-
+        $TYPE_FIELD $MSG_FIELD tty <_log.msg>
+
+    - event.original: >-
+        $TYPE_FIELD $MSG_FIELD <_log.msg>
+
+    - event.original: >-
+        $TYPE_FIELD $MSG_FIELD
+
+    # Logs that start with "node="
+    - event.original: >-
+        $NODE_FIELD $TYPE_FIELD $MSG_FIELD <_log.msg>
+        old auid=<auditd.log.old_auid> new auid=<auditd.log.new_auid>
+        old ses=<auditd.log.old_ses> new ses=<auditd.log.new_ses>
+
+    - event.original: >-
+        $NODE_FIELD $TYPE_FIELD
+        $MSG_FIELD <_log.msg/quoted/ / msg=><_log.sub_msg/quoted/' /'>
+
+    - event.original: >-
+        $NODE_FIELD $TYPE_FIELD
+        $MSG_FIELD <_log.msg/quoted/ / msg=><_log.sub_msg/quoted/" /">
+
+    - event.original: >-
+        $NODE_FIELD $TYPE_FIELD
+        $MSG_FIELD<_log.msg/quoted/ / msg=><_log.sub_msg/quoted/' /'>
+
+    - event.original: >-
+        $NODE_FIELD $TYPE_FIELD
+        $MSG_FIELD<_log.msg/quoted/ / msg=><_log.sub_msg/quoted/" /">
+
+    - event.original: >-
+        $NODE_FIELD $TYPE_FIELD
+        $MSG_FIELD <_log.msg/quoted/ / msg=><_log.sub_msg/quoted/'/'>
+
+    - event.original: >-
+        $NODE_FIELD $TYPE_FIELD
+        $MSG_FIELD <_log.msg/quoted/ / msg=><_log.sub_msg/quoted/"/">
+
+    - event.original: >-
+        $NODE_FIELD $TYPE_FIELD
+        $MSG_FIELD<_log.msg/quoted/ / msg=><_log.sub_msg/quoted/'/'>
+
+    - event.original: >-
+        $NODE_FIELD $TYPE_FIELD
+        $MSG_FIELD<_log.msg/quoted/ / msg=><_log.sub_msg/quoted/"/">
+
+    - event.original: >-
+        $NODE_FIELD $TYPE_FIELD
+        $MSG_FIELD <_avc/quoted/avc:/for > <_log.msg>
+
+    - event.original: >-
+        $NODE_FIELD $TYPE_FIELD
+        $MSG_FIELD <_avc/quoted/avc:/for ><_log.msg>
+
+    - event.original: >-
+        $NODE_FIELD $TYPE_FIELD
+        $MSG_FIELD tty <_log.msg>
+
+    - event.original: >-
+        $NODE_FIELD $TYPE_FIELD $MSG_FIELD <_log.msg>
+
+    - event.original: >-
+        $NODE_FIELD $TYPE_FIELD $MSG_FIELD
+
+normalize:
+  - check:
+      - _log.msg: +exists
+      - _log.sub_msg: +exists
+    map:
+      _kv_msg: +s_concat/$_log.msg/ /$_log.sub_msg
+
+  - check:
+      - _log.sub_msg: +not_exists
+      - _log.msg: +exists
+    map:
+      _kv_msg: $_log.msg
+
+  - map:
+      event.kind: event
+      timestamp: $_log.epoch
+      event.action: +s_lo/$_log.record_type
+      wazuh.avc: +rename_field/$_avc
+      _log: +delete_field
```

Comment:
didn't we decide to leave temporary variables without erasing them? (when correctly implemented should be automatically deleted at the end of the asset)